<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeepSimpleLesson</class>
 <widget class="QStackedWidget" name="KeepSimpleLesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QWidget" name="page">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
     </font>
    </property>
    <property name="text">
     <string>Keep it simple</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>When creating a flag, you want to make sure that your flag is not too complicated. The flag should be so simple that a child can draw it from memory. Flags flap. Flags drape. Flags must be seen from a distance and from their opposite side. Under these circumstances, only simple designs make effective flags. </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_2">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
     </font>
    </property>
    <property name="text">
     <string>Keep it simple</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Furthermore, complicated flags cost more to make, which often can limit how widely they are used. Most poor designs have the elements of a great flag in them—simplify them by focusing on a single symbol, a few colors, large shapes, and no lettering. Avoid the temptation to include a symbol for everybody. Ideally the design will be reversible or at least recognizable from either side. Don’t put a different design on the back.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
